presentation/src/main/java/com/anytypeio/anytype/presentation/widgets/TreeWidgetContainer.kt

```diff
@@ -95,7 +95,8 @@ class TreeWidgetContainer(
                             level = ROOT_INDENT,
                             expanded = paths,
                             path = widget.id + SEPARATOR + widget.source.id + SEPARATOR,
-                            data = data
+                            data = data,
+                            rootLimit = rootLevelLimit
```

Comment:
please put in the PR description exactly how the limits work.

Reply:
plus tests would be nice:)

Reply:
done